plugins/workspace-minimap/test/coordinate_conversion.mocha.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+const chai = require('chai');
+const assert = chai.assert;
+const {Minimap} = require('../src/minimap');
+
+
+suite('Interactions Suite (Shape, Size, Location)', function() {
```

Comment:
Test naming is super important, just like function and variable naming! If a test fails, you want to know immediately what behavior is broken.

For example, the below suite is called "Interactions Suite (Shape, Size, Location) > Square, Medium, Center". I know that you're testing the shape of the content! But someone else dealing with your failing test might not. 

I would probably update the top-level suite to be something like "Click coordinates are properly converted from the minimap to the primary workspace" and the lower-level suite to be something like "Square, medium, and centered content". But go through and figure out what you think works!

Here's a good article about test naming: https://testing.googleblog.com/2014/10/testing-on-toilet-writing-descriptive.html